great-idea-website/css/index.css

```diff
@@ -63,4 +63,68 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+main {
+	width: 900px;
+	display: block;
+	margin: auto;
+}
+header {
+	height: 80px;
+	padding: 5px;
+	margin: auto;
+	vertical-align: middle;
+}
+nav {
+	display: inline-block;
+}
+nav a {
+	text-decoration: none;
+	color: gray;
+	font-size: 15px;
+	font-family: Helvetica;
+	width: 60px;
+	height: 20px;
+	padding: 20px;
+}
+.logo {
+	display: inline-block;
+	padding: 20px;
+	margin: 0px 0px 0px 50px;
+}
+
+h1 {
+	box-sizing: border-box;
+	font-size: 75px;
+}
+
+.tagline {
+	box-sizing: border-box;
+	display: inline-block;
+	text-align: center;
+	width: 379px;
+	height: 379px;
+}
+
+.headerimg {
+	box-sizing: border-box;
```

Comment:
box-sizing : border-box should be on the a parent container, like on the html selector, not on one individual element selector.